code/modules/clothing/neck/mantles.dm

```diff
@@ -0,0 +1,63 @@
+//Mantles!
+
+/obj/item/clothing/neck/mantle
+	name = "mantle"
+	desc = "A heavy quilted mantle, for keeping your shoulders warm and stylish."
+	icon_state = "mantle"
+	item_state = "mantle"
+	body_parts_covered = UPPER_TORSO|ARMS
+	cold_protection = UPPER_TORSO|ARMS
+	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
+
+/obj/item/clothing/neck/mantle/regal
+	name = "regal shawl"
+	desc = "A fancy shawl for nobility, made from high quality materials."
+	icon_state = "regal_mantle"
+	item_state = "regal_mantle"
+
+/obj/item/clothing/neck/mantle/old
+	name = "old wrap"
+	desc = "A tattered fabric wrap, faded over the years. Smells faintly of cigars."
+	icon_state = "old_mantle"
+	item_state = "old_mantle"
+
+//Mantles of the heads
+
+/obj/item/clothing/neck/mantle/captain
+	name = "captain's mantle"
+	desc = "An piece of fashion for the ruling elite. Protect your upper half in style."
+	icon_state = "capmantle"
+	item_state = "capmantle"
+
+/obj/item/clothing/neck/mantle/chief_engineer
+	name = "chief engineer's mantle"
+	desc = "A slick, authoritative cloak designed for the Chief Engineer."
+	icon_state = "cemantle"
+	item_state = "cemantle"
+
+/obj/item/clothing/neck/mantle/chief_medical_officer
+	name = "chief medical officer's mantle"
+	desc = "An absorbent, clean cover found on the shoulders of the Chief Medical Officer."
+	icon_state = "cmomantle"
+	item_state = "cmomantle"
+
+/obj/item/clothing/neck/mantle/head_of_security
+	name = "head of security's shawl"
+	desc = "A shawl worn by the Head of Security. Do you dare take up their mantle?"
+	icon_state = "hosmantle"
+	item_state = "hosmantle"
+	strip_delay = 6 SECONDS
+	put_on_delay = 4 SECONDS
```

Comment:
Since this doesn't actually give any armour anymore, you can probably just remove both of these delays.
```suggestion
	icon_state = "hosmantle"
```